File: reni-backend/src/main/java/pe/gob/mimp/reni/dto/request/CentroAtencionCargaMasivaDetalleRequest.java
package pe.gob.mimp.reni.dto.request;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class CentroAtencionCargaMasivaDetalleRequest implements Serializable {
	private static final long serialVersionUID = 1L;

	private String nombre;
	private String codigo;
	private Date fechaCreacion;
	private String idtTipoCentro;
	private String nomTipoCentro;
	private String idtSubtipoCentro;
	private String nomSubtipoCentro;
	private String ubigeo;
	private String departamento;
	private String provincia;
	private String distrito;
	private String direccion;
	private String refDireccion;
	private String areaResid;
	private Integer flgTieneTelef;
	private Integer flgTieneLuz;
	private Integer flgTieneAgua;
	private Integer flgTieneDesague;
	private Integer capacidadMaxima;
	private Long idTipDocRepres;
	private String nroDocRepres;
	private String nombreRepres;
	private String apePaternoRepres;
	private String apeMaternoRepres;
	private String nroTelefono;
	private String tipoCoordenada;
	private String coordenadaX;
	private String coordenadaY;
	private Integer flgActivo;
	private List<CentroAtencionCargaMasivaServicioRequest> listaServicio;
}

File: reni-frontend/src/main/reni-frontend/src/app/modules/intranet/dto/request/centro-atencion-carga-masiva-detalle.request.ts
import { CentroAtencionCargaMasivaServicioRequest } from "./centro-atencion-carga-masiva-servicio.request";

export class CentroAtencionCargaMasivaDetalleRequest {
    nombre: string;
    codigo: string;
    fechaCreacion: Date;
    idtTipoCentro: string;
    nomTipoCentro: string;
    idtSubtipoCentro: string;
    nomSubtipoCentro: string;
    ubigeo: string;
    departamento: string;
    provincia: string;
    distrito: string;
    direccion: string;
    refDireccion: string;
    areaResid: string;
    flgTieneTelef: number;
    flgTieneLuz: number;
    flgTieneAgua: number;
    flgTieneDesague: number;
    capacidadMaxima: number;
    idTipDocRepres: number;
    nroDocRepres: string;
    nombreRepres: string;
    apePaternoRepres: string;
    apeMaternoRepres: string;
    nroTelefono: string;
    tipoCoordenada: string;
    coordenadaX: string;
    coordenadaY: string;
    flgActivo: number;
    listaServicio: CentroAtencionCargaMasivaServicioRequest[];

    error: any[];
}